go/chat/unfurl/utils.go

```diff
@@ -49,6 +50,11 @@ func GetDomain(uri string) (res string, err error) {
 	if hostname == types.MapsDomain {
 		return hostname, nil
 	}
+	// Check if hostname is an IP address. If so, return it directly
+	// since publicsuffix.EffectiveTLDPlusOne doesn't work with IPs.
+	if net.ParseIP(hostname) != nil {
```

Comment:
https://go-review.googlesource.com/c/net/+/715100/2/publicsuffix/list.go#88 checked IP addresses in the `publicsuffix.PublicSuffix` function which now causes an error to be generated in the `EffectiveTLDPlusOne` call. instead just filter out IP addresses early

cc @mmaxim